COVID Return Policy and Updates (#287)
FarsetLabs/farsetlabs.github.io: about/coronavirus_return_policy.md

- If you are showing any of the known symptoms of COVID-19, please follow government guidelines on self-Isolating, and **do not come to the space**
- **If you become symptomatic or test positive for COVID-19 and have accessed Farset Labs please notify one of the Labs trustees.** It may be necessary to notify other members if there is a chance you have been in close proximity to them. This can be done without disclosing your identity if you wish
- **Wash your hands** as soon as you arrive at Farset Labs and frequently thereafter
- Please **avoid physical contact and maintain at least 2 metres distance** from other Farset Labs members
- **No more than fifteen (15) people are to be in the building at a given time,** and should endeavour to spread themselves out to enforce social distancing.
- Where possible, you should **avoid touching hard surfaces** in communal areas of the building **and should wash or sanitise your hands after doing so**

> - In order to manage the number of people using the building, **non-members are not permitted to access the building.** 

Author: ```suggestion
- In order to manage the number of people using the building, **only members are permitted to access the building.** 
```

# Farset Labs COVID-19 Return Guidelines 

## Purpose

> The purpose of these guidelines is to set out the procedures and restrictions that are put in place to allow Farset Labs to open in a safe manner during the ongoing COVID-19 pandemic

Reviewer: Missing full stop after pandemic

> The purpose of these guidelines is to set out the procedures and restrictions that are put in place to allow Farset Labs to open in a safe manner during the ongoing COVID-19 pandemic

Author: ```suggestion
The purpose of these guidelines is to set out the procedures and restrictions that are put in place to allow Farset Labs to open in a safe manner during the ongoing COVID-19 pandemic.
```

### Spread of COVID-19

From what is known about other coronaviruses, the spread of COVID-19 is most likely to happen when there is close contact with an infected person. It is likely that the risk increases the longer someone has close contact with an infected person. Respiratory secretions produced when an infected person coughs or sneezes containing the virus are most likely to be the main means of transmission.

There are 2 main routes by which people can spread COVID-19:

- infection can be spread to people who are nearby (within 2 metres) or possibly could be inhaled into the lungs
- it is also possible that someone may become infected by touching a surface, object or the hand of an infected person that has been contaminated with respiratory secretions and then touching their own mouth, nose, or eyes (such as touching contaminated surfaces (e.g. door knob) or shaking hands then touching own face)

## Organisational Named Person For COVID-19 Response

Best practices dictate that organisations should have a named person responsible for their response to the COVID-19 pandemic.  

> As such, Andrew Bolster will be responsible for these guidelines, will be the contact person for questions regarding our response to COVID-19 and will be responsible for organisational contact tracing in the event of a member reporting an infection. ([bolster@farsetlabs.org.uk](mailto:bolster@farsetlabs.org.uk) / 07783249547).

Author: ```suggestion
As such, Art Knipe will be responsible for these guidelines, will be the contact person for questions regarding our response to COVID-19 and will be responsible for organisational contact tracing in the event of a member reporting an infection. ([art@farsetlabs.org.uk](mailto:art@farsetlabs.org.uk)).
```

> Should Andrew be unable to fulfil this role for any reason Colin Mitchell, will deputise ([colin.mitchell@farsetlabs.org.uk](mailto:colin.mitchell@farsetlabs.org.uk)) 

Reviewer: unnecessary comma after Colin's name

> Should Andrew be unable to fulfil this role for any reason Colin Mitchell, will deputise ([colin.mitchell@farsetlabs.org.uk](mailto:colin.mitchell@farsetlabs.org.uk)) 

Author: ```suggestion
Should Artbe unable to fulfil this role for any reason Colin Mitchell will deputise ([colin.mitchell@farsetlabs.org.uk](mailto:colin.mitchell@farsetlabs.org.uk)) 
```

## Guidelines for Reopening of Farset Labs

> Farset Labs will begin a phased reopening on 22/06/20 at 10am. The reopening will proceed based on the following guidance:

Author: ```suggestion
Farset Labs will begin a phased reopening on 24/06/20 at 10am. The reopening will proceed based on the following guidance:
```

- Upon reopening we will request that our members continue to work from home if they can, in line with the [guidance from the NI Executive](https://www.executiveoffice-ni.gov.uk/sites/default/files/publications/execoffice/execuitveour-approach-to-decision-making.pdf). 
- Access to the building will be subject to these new guidelines.
- It will be left to the discretion and good sense of individual members to decide whether they can continue to work from home or need to use the space.
- All members will be required to complete and sign an online Pre-Return form which indicates their agreement with these guidelines. This must be done before they can return to the space.

> If you wish to return to the space at this time, you must contact Andrew Bolster by email ([bolster@farsetlabs.org.uk](mailto:bolster@farsetlabs.org.uk)) to notify him of your intention to return. You will then be asked to fill in a Pre-Return Disclaimer, after which you will be given new access control codes.

Author: ```suggestion
If you wish to return to the space at this time, you must contact Art Knipe by email ([art@farsetlabs.org.uk](mailto:art@farsetlabs.org.uk)) to notify him of your intention to return. You will then be asked to fill in a Pre-Return Disclaimer, after which you will be given new access control codes.
```

**Do not return to the space if you have not already digitally signed the pre-return disclaimer as you will not be permitted in the building**

## Guidelines For Access To Farset Labs During Ongoing Covid-19 Restrictions

> It is anticipated that social distancing restrictions will remain in place for some time. In order to respect these restrictions and to ensure the levels of cleanliness required to prevent infection members will be expected to adhere to the following guidelines.

Reviewer: should be a colon at the end instead of a full stop

> It is anticipated that social distancing restrictions will remain in place for some time. In order to respect these restrictions and to ensure the levels of cleanliness required to prevent infection members will be expected to adhere to the following guidelines.

Author: ```suggestion
It is anticipated that social distancing restrictions will remain in place for some time. In order to respect these restrictions and to ensure the levels of cleanliness required to prevent infection members will be expected to adhere to the following guidelines;
```
